### ISSUE TITLE
Profiles table rows should have role "checkbox" and check on space

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#45841]
The profiles table rows should have the accessibility role "checkbox"  and pressing "space" should toggle the checkbox. Similar how breakpoints list works.

![Image](https://github-com/microsoft/vscode/assets/1926584/871c4de6-71c0-4844-9dec-5e7bffee6286)
